data/scripts/getSTVGeoJSON.sh

```diff
@@ -1,33 +1,30 @@
-#!/bin/bash
+#!/bin/sh
 
 # Chron.
 # Copyright (C) 2019 Alisa Belyaeva, Ata Ali Kilicli, Amaury Martiny,
 # Daniil Mordasov, Liam O’Flynn, Mikhail Orlov.
-# 
+#
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation, either version 3 of the License, or
 # (at your option) any later version.
-# 
+#
 # This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
-# 
+#
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-psql='psql -U postgres -t -c'
+psql="psql -d \"host=db user=${POSTGRES_USER} password=${POSTGRES_PASSWORD}\" -t -c"
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+DIR="$( cd "$( dirname "$0" )" && pwd )"
 DATA="${DIR}/../data"
 mkdir -p $DATA
 LAYER='stv'
 FCFILE="${DATA}/${LAYER}.json"
 
```

Comment:
I would propose to make some kind of lock, to eliminate possibility of the race condition as this script can be invoked via crond and through Makefile call